src/KintoID.sol

```diff
@@ -57,15 +57,21 @@ contract KintoID is
     /// state-changing operation, so as to prevent replay attacks, i.e. the reuse of a signature.
     mapping(address => uint256) public override nonces;
 
-    bytes32 public domainSeparator;
+    bytes32 public override domainSeparator;
+
+    // This mapping is used to enable recovery transfer
+    mapping(address => address) public override enabledRecoveryTransfer;
+
+    address public immutable override walletFactory;
 
     /* ============ Modifiers ============ */
```

Comment:
we could remove this since there's no modifier

Reply:
done